src/libstd/collections/mod.rs

```diff
@@ -309,7 +309,7 @@
 //! }
 //! ```
 
-#![experimental]
+#![stable]
 
```

Comment:
Should we not be marking some of the facade modules made in this file (and/or libcollections/lib.rs) as #stable?